src/jv.c

```diff
@@ -1772,7 +1777,12 @@ jv jv_object_set(jv object, jv key, jv value) {
   // copy/free of object, key, value coalesced
   jv* slot = jvp_object_write(&object, key);
   jv_free(*slot);
-  *slot = value;
+  if(jv_is_borrowed(value)){
```

Comment:
Can this not be written as `*slot = jv_unborrow(value)` because that's the same as `*slot = value` when `value` is not borrowed?

Reply:
but it could be replaced with jv_return() (which does not copy owned values)